lib/lightning_web/controllers/credential_transfer_controller.ex

```diff
@@ -0,0 +1,35 @@
+defmodule LightningWeb.CredentialTransferController do
+  use LightningWeb, :controller
+
+  alias Lightning.Credentials
+
+  def confirm(conn, %{
+        "token" => token,
+        "receiver_id" => receiver_id,
+        "credential_id" => credential_id
+      }) do
+    %{assigns: %{current_user: %{id: owner_id}}} = conn
```

Comment:
For transfering a credential seems to be the most responsible and safe way by requiring the user to be authenticaed.

Reply:
That's is already done in the Router

```elixir
pipe_through [:browser, :require_authenticated_user]
```

Reply:
Sure, I was praising your careful approach (:

Reply:
Ah ! Thanks my brother 😊